### PR TITLE
add multiple result comparison for plots

### PR DESCRIPTION
**Pull request by Lilyhanig**

### CODE REVIEW

File: PowerSimulationsPlots/src/plot_recipes.jl [hunk 1]
```diff
@@ -3,6 +3,21 @@ import PlotlyJS
 import WebIO
 WebIO.install_jupyter_nbextension()
 
+function _format_multiple_plots(length::Int, plots::Array)
```
Comment (daniel-thom) [inline PowerSimulationsPlots/src/plot_recipes.jl]: Why is this function necessary?  Does `plots` have extra entries that you are truncating?  If so, could you write
```plots = plots[1:length]```

File: PowerSimulationsPlots/src/call_plots.jl [hunk 2]
```diff
@@ -78,6 +78,26 @@ PLOTLY_DEFAULT = vcat(
     CURTAILMENT_288,
 )
 
+function _filter_variables(results::PSI.Results; kwargs...)
+    filter_results = Dict()
+    reserves = get(kwargs, :reserves, false)
+    if reserves
+        for (key, var) in results.variables
+            if "$key"[1:2] == "P_" || "$key"[1:5] == "Spin_" || "$key"[1:4] == "Reg_" || "$key"[1:5] == "Flex_"
```
Comment (daniel-thom) [inline PowerSimulationsPlots/src/call_plots.jl]: This feels brittle. Are these strings coming from PSI const definitions for variables?  Seems like you could split by underscore and then compare the values.  And that could be made part of PSI.

File: PowerSimulationsPlots/src/plot_recipes.jl (hunk 1 above)
Comment (Lilyhanig) [reply PowerSimulationsPlots/src/plot_recipes.jl]: couldn't figure this out for now....

Comment (jd-lara) [reply PowerSimulationsPlots/src/plot_recipes.jl]: Add a comment on why the hacky workaround.

**Pull request merged**